Scripts/detect_public_api_changes/package_file_helper.swift

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env xcrun swift
+
+import Foundation
+
+// MARK: - Convenience Methods
+
+/// Extracts all targets in the targets section
+///
+/// Returns: All available target names
+func availableTargets(from packageContent: String) -> [String] {
+    let scanner = Scanner(string: packageContent)
+    _ = scanner.scanUpToString("targets: [")
+
+    var availableTargets = Set<String>()
+
+    while scanner.scanUpToString(".target(") != nil {
+        let nameStartTag = "name: \""
+        let nameEndTag = "\""
+        
+        _ = scanner.scanUpToString(nameStartTag)
+        _ = scanner.scanString(nameStartTag)
+        
+        if let targetName = scanner.scanUpToString(nameEndTag) {
+            availableTargets.insert(targetName)
+        }
+    }
+    
+    return availableTargets.sorted()
+}
+
+/// Generates a library entry from the name and available target names to be inserted into the `Package.swift` file
```

Comment:
Nice!
However, for someone who has spent not enough time researching how the api digester works it might be a bit overwhelming to understand how every small step consolidates in the big picture.

Reply:
Will write some proper documentation and other findings once/before the [command line version](https://github.com/Adyen/adyen-ios/pull/1718) is done